File: features/digital/digitalproduct/src/main/java/com/tokopedia/digital/widget/data/entity/response/Relationships.java
package com.tokopedia.digital.widget.data.entity.response;

import com.google.gson.annotations.SerializedName;
import com.tokopedia.digital.common.data.entity.response.Category;
import com.tokopedia.digital.common.data.entity.response.RelationData;

/**
 * @author rizkyfadillah on 10/2/2017.
 */

public class Relationships {

    @SerializedName("category")
    private Category category;

    @SerializedName("operator")
    private Operator operator;

    public Category getCategory() {
        return category;
    }

    public Operator getOperator() {
        return operator;
    }

    public class Operator {

        @SerializedName("data")
        private RelationData data;

        public RelationData getData() {
            return data;
        }
    }

}

File: features/recharge/digitalproduct/src/main/java/com/tokopedia/digital/newcart/data/entity/requestbody/checkout/Relationships.kt
package com.tokopedia.digital.newcart.data.entity.requestbody.checkout

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * @author anggaprasetiyo on 3/9/17.
 */

class Relationships(@field:SerializedName("cart")
                    @field:Expose
                    var cart: Cart?) {
}
